test/functional/core/job_spec.lua

```diff
@@ -23,6 +23,9 @@ local expect_msg_seq = helpers.expect_msg_seq
 local pcall_err = helpers.pcall_err
 local Screen = require('test.functional.ui.screen')
 
+local run_test = false
+
+if run_test then
```

Comment:
This block is now never run, even when not on FreeBSD?

Reply:
These tests are flaky on all platforms, just way more on freebsd.

Reply:
I know. But these tests are just flaky. They do not always fail.

Reply:
What is your proposal?

Reply:
Since only sourcehut check cannot be restarted without pushing again, I think it is better to only skip these on sourcehut.

Reply:
OK, sounds good.